jobs/mongodb_migration/src/mongodb_migration/migrations/_20230309141600_cache_add_job_runner_version.py

```diff
@@ -0,0 +1,35 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright 2022 The HuggingFace Authors.
+
+import logging
+
+from libcommon.simple_cache import CachedResponse
+from mongoengine.connection import get_db
+
+from mongodb_migration.check import check_documents
+from mongodb_migration.migration import Migration
+
+
+# connection already occurred in the main.py (caveat: we use globals)
+class MigrationAddJobRunnerVerionToCacheResponse(Migration):
```

Comment:
It could be the opportunity to start using tests. This migration changes all the cache entries. It has the potential to break things!

WDYT? The idea would be to extract a small dump (ideally shuffled) of the database, version it in the code, and then, in the test, import it to the test database, launch the test, and check if the migration worked as expected.

Reply:
Yes, I think I can work on it

Reply:
Done but only with one record, simulating worker_version values and also missing value